app/tests/spec/views/elements/coppa-age-input.js

```diff
@@ -61,6 +61,12 @@ define(function (require, exports, module) {
         });
       });
 
+      describe('invalid', () => {
+        it('if age is greater than 130, throws a `INVALID_AGE`', () => {
```

Comment:
Can you also add a test that ensure the boundary condition succeeds, i.e., that `130` does not throw an error?